builtin/providers/google/resource_compute_target_pool.go

```diff
@@ -91,7 +91,12 @@ func resourceComputeTargetPool() *schema.Resource {
 func convertStringArr(ifaceArr []interface{}) []string {
 	arr := make([]string, len(ifaceArr))
 	for i, v := range ifaceArr {
-		arr[i] = v.(string)
```

Comment:
this is a rare case where ignoring the `ok` is useful: `arr[i], _ = v.(string)` replaces the if-else